sap-file-server-md/azuredeploy.json

```diff
@@ -0,0 +1,493 @@
+      },
+      "Linux": {
+        "script": "[concat(variables('github'), 'noop.sh')]",
+        "scriptCall": "sh noop.sh"
+      }
+    },
+    "vmSizes": {      
+        "nfsserversize": "Standard_DS2_v2",
+        "nfsservercount": 2,
+        "nfsserverdisks": [
+          {
+            "size": 128
+          }
+        ]
+    },
+    "internalExtensionScriptCL": "[variables('csExtension')[variables('internalOSType')].script]",
+    "internalExtensionScriptCallCL": "[variables('csExtension')[variables('internalOSType')].scriptCall]",
+    "internalExtensionScriptCallArgsCL": "",
+    "nfsvmSize": "[variables('vmSizes').nfsserversize]",
+    "nfsvmCount": "[variables('vmSizes').nfsservercount]",
+    "nfsvmDataDisks": "[variables('vmSizes').nfsserverdisks]",
+    "sidlower": "[toLower(parameters('resourcePrefix'))]",
+    "vmName": "[variables('sidlower')]",
+    "vnetName": "[concat(variables('sidlower'), '-vnet')]",
+    "publicIPAddressType": "Dynamic",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "Subnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "subnets": {
+      "new": "[concat(resourceId('Microsoft.Network/virtualNetworks', variables('vnetName')), '/subnets/', variables('subnetName'))]",
+      "existing": "[parameters('subnetId')]"
+    },
+    "selectedSubnetId": "[variables('subnets')[parameters('newOrExistingSubnet')]]",
+    "nestedDeploymentName": "nestedTemplate",
+    "nestedDeploymentNameVNet": "[concat(variables('nestedDeploymentName'), 'vnet')]",
+    "nestedDeploymentNamePIP": "[concat(variables('nestedDeploymentName'), 'pip')]",
+    "nestedDeploymentNameCL": "[concat(variables('nestedDeploymentName'), 'nfs')]",
+    "nestedDeploymentNameProf": "[concat(variables('nestedDeploymentName'), 'prof')]",
+    "nestedDeploymentNameNIC": "[concat(variables('nestedDeploymentName'), 'nic')]",
+    "nestedDeploymentNameNICCL": "[concat(variables('nestedDeploymentNameNIC'), 'nfs')]",
+    "nestedDeploymentNameLBCL": "[concat(variables('nestedDeploymentName'), 'lbnfs')]",
+    "publicIpNameCL": "[concat(variables('sidlower'), '-pip-nfs')]",
+    "avSetNameCL": "[concat(variables('sidlower'), '-avset-nfs')]",
+    "nestedDeploymentNameNSG": "[concat(variables('nestedDeploymentNameCL'), 'nsg')]",
+    "nsgNameCL": "[concat(variables('sidlower'), '-nsg')]",
+    "internalStorageTypeCL": "Premium_LRS",
+    "loadBalancerNameCL": "[concat(variables('sidlower'), '-lb-nfs')]",
+    "vmNameCL": "[concat(variables('vmName'), '-nfs')]",
+    "nicNameCL": "[concat(variables('sidlower'), '-nic-nfs')]",
+    "osDiskType": "image"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-01-01",
+      "name": "[variables('nestedDeploymentNameNSG')]",
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('github'), parameters('newOrExistingSubnet'), 'nsg.json')]",

```

Comment:
this is an old pattern - use the condition property on the resources instead - same for the others below